cvmfs/fuse_remount.cc

```diff
@@ -34,6 +35,16 @@ FuseRemounter::Status FuseRemounter::Check() {
     return kStatusMaintenance;
 
   LogCvmfs(kLogCvmfs, kLogDebug, "catalog TTL expired, remount");
+
+  LogCvmfs(kLogCvmfs, kLogDebug, "checking revision against blacklists");
+  if (mountpoint_->ReloadBlacklists() &&
+      mountpoint_->catalog_mgr()->IsRevisionBlacklisted())
+  {
+    LogCvmfs(kLogCatalog, kLogDebug | kLogSyslogErr,
+            "repository revision blacklisted, aborting");
+    abort();
+  }
+
```

Comment:
I think there should probably be a debug message here indicating that the blacklist check is finished and that the remount is about to happen.  Maybe the message at the beginning should just say "catalog TTL expired" (or be incorporated into the checking... message) and a message here that says "remounting".